src/hiero_sdk_python/query/transaction_record_query.py

```
@@ -16,13 +16,16 @@ class TransactionRecordQuery(Query):
     """
 
-    def __init__(self, transaction_id: Optional[TransactionId] = None):
+    def __init__(self, transaction_id: Optional[TransactionId] = None, include_duplicates: bool = False):
         """
         Initializes the TransactionRecordQuery with the provided transaction ID.
         """
         super().__init__()
         self.transaction_id : Optional[TransactionId] = transaction_id
-        
+        self.include_duplicates: bool = bool(include_duplicates)
+    def set_include_duplicates(self, include: bool) -> 'TransactionRecordQuery':
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)